src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: $card-width;
+  padding: $card-padding;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: $black-color;
+
+  &:not(:nth-child(4n)) {
+    margin-right: $column-gap;
+  }
+
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: $row-gap;
```

Comment:
Follow the checklist, please 
<img width="1063" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/99403198/5ca5f502-ee63-4683-b9c7-ec7563272fd5">